docs/croissant-spec.md

```diff
@@ -196,20 +226,17 @@ specifies files to exclude from the `FileSet`. If multiple values are provided,
 then their union is taken, i.e., any file matched by any of the exclude patterns
 is excluded.
 
-**range**:	[sc:Text](https://schema.org/Text)
-
-**domain**:	[FileSet](#fileset)
+**range**: [sc:Text](https://schema.org/Text)
 
+**domain**: [FileSet](#fileset)
 
 ### source
 
-The source of the data for a `RecordSet` or `Field`. In the case of a
-`RecordSet`, this can be one or more `FileObject`, `FileSet` or `RecordSet`. In
-the case of a `Field`, this can be an addressable subset of the contents of a
-`FileObject` or `FileSet` (e.g., the name of a column in a CSV file, or a known
-property of a `FileSet` such as `"filename"` or `"content"`).
-
-**range**:	[Reference](#reference), [DataSource](#datasource)
+The source of the data for a `Field`. In the case of a `RecordSet`, this can be
```

Comment:
or a `RecordSet`? If not, I don't understand why we don't delete "In the case of a `RecordSet`, ..."

Reply:
Done.